livingatlas/pipelines/pom.xml

```diff
@@ -280,7 +280,7 @@
 
     <!-- ALA libraries -->
     <ala-namematching-service.version>1.0-SNAPSHOT</ala-namematching-service.version>
-    <ala-sensitive-data-service.version>1.0-SNAPSHOT</ala-sensitive-data-service.version>
+    <ala-sensitive-data-service.version>1.1-SNAPSHOT</ala-sensitive-data-service.version>
```

Comment:
Just a side comment - perhaps we should start doing proper releases of both `ala-namematching-service` and `ala-sensitive-data-service` and refer to non snapshot version in pipelines.